### PR TITLE
Automate hobgoblin feats

### PR DESCRIPTION
**Pull request by Shandyan**

### CODE REVIEW

File: packs/data/feats.db/vigorous-health.json [hunk 1]
```diff
@@ -11,14 +11,15 @@
             "value": null
         },
         "description": {
-            "value": "<p>Your physique is robust and can withstand blood loss startlingly well. Whenever you would gain the @Compendium[pf2e.conditionitems.Drained]{Drained}condition, you can attempt a DC 17 flat check. On a success, you don't gain the drained condition.</p>"
+            "value": "<p>Your physique is robust and can withstand blood loss startlingly well. Whenever you would gain the @Compendium[pf2e.conditionitems.Drained]{Drained} condition, you can attempt a @Check[type:flat|dc:17]{flat check}. On a success, you don't gain the drained condition.</p>"
```
Comment (SpartanCPA) [inline packs/data/feats.db/vigorous-health.json:14]:
```suggestion
            "value": "<p>Your physique is robust and can withstand blood loss startlingly well. Whenever you would gain the @Compendium[pf2e.conditionitems.Drained]{Drained} condition, you can attempt a @Check[type:flat|dc:17]. On a success, you don't gain the drained condition.</p>"
```
Should be no need for the bracket, it should just display "DC 17 Flat Check" by default.

File: packs/data/feats.db/stone-face.json [hunk 2]
```diff
@@ -19,11 +19,35 @@
...
         },
-        "rules": [],
+        "rules": [
+            {
+                "hideIfDisabled": "true",
+                "key": "FlatModifier",
+                "predicate": {
+                    "any": [
+                        "action:demoralize"
+                    ]
```
Comment (SpartanCPA) [inline packs/data/feats.db/stone-face.json:33]:
```suggestion
                    "any": [
                        "action:demoralize",
                        "action:coerce"
                    ]
```

Might as well tag both intimidation skill actions.

File: packs/data/feats.db/agonizing-rebuke.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
             "value": null
         },
         "description": {
-            "value": "<p>When you terrorize your enemies, you also cause them painful mental distress. When you successfully @Compendium[pf2e.actionspf2e.Demoralize]{Demoralize} a foe, that foe takes 1d4 mental damage at the start of each of its turns as long as it remains @Compendium[pf2e.conditionitems.Frightened]{Frightened} and continues to engage in combat with you. If you have master proficiency in Intimidation, the damage increases to 2d4, and if you have legendary proficiency, the damage increases to 3d4.</p>"
+            "value": "<p>When you terrorize your enemies, you also cause them painful mental distress. When you successfully @Compendium[pf2e.actionspf2e.Demoralize]{Demoralize} a foe, that foe takes [[/r {1d4}[mental]]]{1d4 mental damage} at the start of each of its turns as long as it remains @Compendium[pf2e.conditionitems.Frightened]{Frightened} and continues to engage in combat with you. If you have master proficiency in Intimidation, the damage increases to [[/r {2d4}[mental]]]{2d4}, and if you have legendary proficiency, the damage increases to [[/r {3d4}[mental]]]{3d4}.</p>"
```
Comment (SpartanCPA) [inline packs/data/feats.db/agonizing-rebuke.json:14]: Add a note to Intimidation on success for `action:demoralize`

Comment (Shandyan) [reply packs/data/feats.db/agonizing-rebuke.json:14]: Done and updated re-en.json to match

**Review by SpartanCPA — APPROVED** [review]

**Pull request merged**